src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -118,13 +119,15 @@ public HttpCacheProxyFactory(
         LongSupplier supplyTrace,
         ToIntFunction<String> supplyTypeId,
         SignalingExecutor executor,
-        LongObjectBiConsumer<Runnable> scheduler)
+        LongObjectBiConsumer<Runnable> scheduler,
+        int preferWaitMaximum)
```

Comment:
Suggest passing in the `HttpCacheConfiguration config` as first parameter to the constructor, then extracting the `preferWaitMaximum` and assigning in the constructor implementation.

This will scale better as we add more configuration items without making the constructor parameter list longer.

Reply:
Done.